packages/devtools_app/lib/src/shared/table/table.dart

```diff
@@ -1597,6 +1620,38 @@ class _TableRowState<T> extends State<TableRow<T>>
   bool shouldShow() => widget.isShown;
 }
 
+class _RowGuidelinePainter extends CustomPainter {
+  _RowGuidelinePainter(this.level, this.colorScheme);
+
+  final int level;
+  final ColorScheme colorScheme;
```

Comment:
nit: pass in the list of colors directly since the ColorScheme isn't used for anything else